packages/toolpad-core/src/context/BrandingContext.ts

```diff
@@ -0,0 +1,6 @@
+import * as React from 'react';
```

Comment:
I think we should try to avoid grouping things by function (e.g. "contexts", "modules", "types", "controllers", "views"). That doesn't scale.
If it's internal only, I'd nest it under the `AppProvider`. If we will export this as an API, I think we can do like base does it, under a folder named as the hook that will eventually consume this context: https://github.com/mui/base-ui/tree/master/packages/mui-base/src/useTabs